src/porting.cpp

```diff
@@ -920,22 +920,34 @@ double perf_freq = get_perf_freq();
  *
  * As a workaround we track freed memory coarsely and call malloc_trim() once a
  * certain amount is reached.
+ *
+ * In situations with high "throughput" of allocated/freed memory this can cause
+ * jitter (in particular on the client), so we additionally restrict this to only
+ * happen once a minute.
  */
 
 static std::atomic<size_t> memory_freed;
 
-constexpr size_t MEMORY_TRIM_THRESHOLD = 128 * 1024 * 1024;
+static u64 next_trim_time;
+
+constexpr size_t MEMORY_TRIM_THRESHOLD = 256 * 1024 * 1024;
```

Comment:
The size_t could overflow in a minute, right?

Reply:
`size_t` is unsigned, so goes up to 2^32 - 1 or 2^64 - 1 respectively, depending on how large a result of `sizeof` can be (32- or 64-bit machine), which is an upper bound on the amount of memory that can be allocated, so I don't see how this could possibly overflow.